content/references/rippled-api/transaction-formats/transaction-results/transaction-results.ja.md

```diff
@@ -1,8 +1,8 @@
-# 取引の結果
+# トランザクションの結果
```

Comment:
This is just a weird thing I noticed while I was working on translating the send-xrp tutorial; this page ("Transaction Results") translates "Transaction" as 取引 instead of トランザクション, which is inconsistent with the usage on most of the rest of the site. So I fixed it.